Sources/Layouts/BaseLayout.swift

```diff
@@ -30,10 +30,21 @@ open class BaseLayout<V: View> {
         return config != nil
     }
 
+    private let viewClass: View.Type
```

Comment:
The reason for this is objective-c doesn't support `V`?

Reply:
Objective C has some support for generics but it's mostly for defining neat interfaces for Swift, and some mild type checking on Obj C side, not nearly as full featured as Swift generics.